lms/djangoapps/mobile_api/testutils.py

```diff
@@ -10,7 +10,7 @@
      MobileCourseAccessTestMixin - tests for APIs with mobile_course_access and verify_enrolled=False.
      MobileEnrolledCourseAccessTestMixin - tests for APIs with mobile_course_access and verify_enrolled=True.
 """
-# pylint: disable=no-member
+# pylint: disable=no-member, invalid-name
```

Comment:
Shouldn't this be disabled on the line itself (line 143 as of right now)? It occurs once.